starter-code/src/App.js

```diff
@@ -1,25 +1,25 @@
 import React, { Component } from 'react';
 import logo from './logo.svg';
 import './App.css';
+import CountryDet from './components/CountryDetail.js';
+import {countryList} from './components/CountriesDet';
```

Comment:
Try to stay consistent with the naming of your components, so that the export default as well as the imports correspond to the name of your .js files.